src/common.h

```diff
@@ -244,6 +244,28 @@ typedef struct TypInputFile TypInputFile;
 typedef struct TypOutputFile TypOutputFile;
 
 
+/****************************************************************************
+**
+*T  ExecStatus . . . .  type of status values returned by read, eval and exec
+**                      subroutines, explaining why evaluation, or execution
+**                      has terminated.
+**
+**  Values are powers of two; this is used to test a given status for
+**  multiple possible values simultaneously.
```

Comment:
this statement isn't true?

Reply:
on reading, I believe the plan is to remove this 'power of two' and not do bit-oring tricks (which I agree are better to get rid of), so it's just this comment that needs removing?

Reply:
those two lines are removed in the first patch and then accidentally re-introduced in the second. I'll get rid of them.

Reply:
Fixed now, thanks for pointing this out!